libs/designer/src/lib/ui/configuretemplate/workflows/selectWorkflows.tsx

```diff
@@ -0,0 +1,218 @@
+import { ResourcePicker } from '../../templates';
+import { useSelector } from 'react-redux';
+import { useCallback } from 'react';
+import { equals, type WorkflowResource, type LogicAppResource } from '@microsoft/logic-apps-shared';
+import {
+  TableBody,
+  TableCell,
+  TableRow,
+  Table,
+  TableHeader,
+  TableHeaderCell,
+  TableSelectionCell,
+  TableCellLayout,
+  useTableFeatures,
+  type TableColumnDefinition,
+  useTableSelection,
+  createTableColumn,
+  Skeleton,
+  SkeletonItem,
+} from '@fluentui/react-components';
+import { useResourceStrings } from '../resources';
+
+export const SelectWorkflows = ({
+  selectedWorkflowsList,
+  onWorkflowsSelected,
+}: {
+  selectedWorkflowsList: Record<string, Partial<WorkflowResource>>;
+  onWorkflowsSelected: (normalizedWorkflowIds: string[]) => void;
+}) => {
+  const intl = useIntl();
+  const { isConsumption, logicAppName, subscriptionId, resourceGroup } = useSelector((state: RootState) => ({
+    isConsumption: !!state.workflow.isConsumption,
+    logicAppName: state.workflow.logicAppName,
+    subscriptionId: state.workflow.subscriptionId,
+    resourceGroup: state.workflow.resourceGroup,
+    workflowsInTemplate: state.template.workflows,
+    selectedTabId: state.tab.selectedTabId,
+  }));
+  const { data: workflows, isLoading } = useWorkflowsInApp(subscriptionId, resourceGroup, logicAppName ?? '', !!isConsumption);
+
+  const onLogicAppSelected = useCallback(
+    (app: LogicAppResource) => {
+      const { id, plan } = app;
+      if (equals(plan, 'Consumption')) {
+        const normalizedWorkflowId = id.toLowerCase();
+        onWorkflowsSelected([normalizedWorkflowId]);
+      } else {
+        onWorkflowsSelected([]);
+      }
+    },
+    [onWorkflowsSelected]
+  );
+
+  const resourceStrings = useResourceStrings();
+
+  const intlText = {
+    SOURCE: intl.formatMessage({
+      defaultMessage: 'Source',
+      id: '3LM7R3',
+      description: 'Title for the resource selection section',
+    }),
+    SOURCE_LABEL: intl.formatMessage({
+      defaultMessage: 'Select the Logic App service you would like to add workflows from.',
+      id: 'CLb9Hv',
+      description: 'Label for the logic app service resource selection description',
+    }),
+    WORKFLOWS: intl.formatMessage({
+      defaultMessage: 'Workflows',
+      id: 'NHuGg3',
+      description: 'Title for the workflows selection section',
+    }),
+    WORKFLOWS_LABEL: intl.formatMessage({
+      defaultMessage: 'Select the workflows you would like to add to this template.',
+      id: 'YGe6mJ',
+      description: 'Label for the workflows selection description',
+    }),
+    WORKFLOW_NAME: intl.formatMessage({
+      defaultMessage: 'Name',
+      id: 'kLqXDY',
+      description: 'Label for workflow Name',
+    }),
+    TRIGGER_TYPE: intl.formatMessage({
+      defaultMessage: 'Trigger',
+      id: 'mXwGRH',
+      description: 'Label for workflow trigger type',
+    }),
```

Comment:
These seem to be getting used in other places isnt this already present in the resources?